test/workloadapi/test_default_workload_api_client_jwt.py

```diff
@@ -115,36 +112,59 @@ def test_fetch_jwt_svid_wrong_token(mocker):
         )
     )
     with pytest.raises(FetchJwtSvidError) as exception:
-        client.fetch_jwt_svid(audiences=audience)
+        WORKLOAD_API_CLIENT.fetch_jwt_svid(audiences=set(audience))
 
     assert str(exception.value).startswith('Error fetching JWT SVID')
 
 
 def test_fetch_jwt_svid_no_token_returned(mocker):
-    client = DefaultWorkloadApiClient('unix:///dummy.path')
     audience = ['spire', 'test', 'valid']
-    client._spiffe_workload_api_stub.FetchJWTSVID = mocker.Mock(
+    WORKLOAD_API_CLIENT._spiffe_workload_api_stub.FetchJWTSVID = mocker.Mock(
         return_value=iter([workload_pb2.JWTSVIDResponse(svids=[])])
     )
     with pytest.raises(FetchJwtSvidError) as exception:
-        client.fetch_jwt_svid(audiences=audience)
+        WORKLOAD_API_CLIENT.fetch_jwt_svid(audiences=set(audience))
 
     assert str(exception.value).startswith('Error fetching JWT SVID')
 
 
 """
-
-
 # TODO: Implement using WorkloadApi Mock
 def test_fetch_jwt_bundles():
     wlapi = get_client()
     wlapi.fetch_jwt_bundles()
+"""
 
-# TODO: Implement using WorkloadApi Mock
-def test_validate_jwt_svid():
-    wlapi = get_client()
-    token = 'TODO'
-    audiences = 'foo'
-    wlapi.validate_jwt_svid(token=token, audience=audiences)
 
-"""
+def test_validate_jwt_svid(mocker):
+    audience = 'spire'
+    spiffe_id = 'spiffe://test.com/my_service'
+    jwt_svid = create_jwt(rsa_key_pem, 'kid1', 'RS256', [audience], spiffe_id)
+
+    WORKLOAD_API_CLIENT._spiffe_workload_api_stub.ValidateJWTSVID = mocker.Mock(
+        return_value=workload_pb2.ValidateJWTSVIDResponse(
+            spiffe_id=spiffe_id,
+        )
+    )
+
+    svid = WORKLOAD_API_CLIENT.validate_jwt_svid(token=jwt_svid, audience=audience)
+
+    assert svid.spiffe_id == SpiffeId.parse(spiffe_id)
+    assert svid.token == jwt_svid
+    assert svid.claims['aud'] == [audience]
+    assert svid.audience == {audience}
+
+
+def test_validate_jwt_svid_raise_error(mocker):
+    audience = 'spire'
+    spiffe_id = 'spiffe://test.com/my_service'
+    jwt_svid = create_jwt(rsa_key_pem, 'kid1', 'RS256', [audience], spiffe_id)
```

Comment:
in this case, you do not need any of these parameters as there is no validation in the method itself.....